src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { ApiError, defaultErrorTexts, ErrorType } from "../api/axios";
+import Alertstripe from "nav-frontend-alertstriper";
+import { Normaltekst } from "nav-frontend-typografi";
+import Lenke from "nav-frontend-lenker";
+import styled from "styled-components";
+
+const texts = {
+  meldFeil: "Meld oss gjerne om feilen her",
+};
+
+const newJiraTicketUrl =
+  "https://jira.adeo.no/plugins/servlet/desk/portal/541/create/1401";
+
+const InlineLenke = styled(Lenke)`
+  margin-left: 0.2em;
+`;
+
+const textWithJiraLink = (text: string) => {
+  return (
+    <>
+      <span>{text}</span>
+      <InlineLenke target="_blank" href={newJiraTicketUrl}>
+        {texts.meldFeil}
+      </InlineLenke>
+    </>
+  );
+};
+
+interface ErrorBoundaryProps {
+  apiError?: ApiError | null;
+  errorMessage?: string;
+}
+
+type State = {
+  hasError: boolean;
+};
+
+class ErrorBoundary extends React.Component<ErrorBoundaryProps, State> {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  componentDidCatch() {
```

Comment:
Visste ikke om denne lifecycle-hooken. Så det finnes ingen rett fram måte å lage ErrorBoundary som funksjonell komponent med hooks? 🤯

Reply:
Nei, må ha class for å få med den componentDidCatch saken :(